### DISCUSSION TITLE
How to Patch a Font

### DISCUSSION THREAD

**Discussion by waydegilliam** [ryanoasis/nerd-fonts#878] [Show and tell]
Hello!

As someone who had struggled using the Nerd Fonts `font-patcher` script, I wrote a [tool](https://github.com/waydegg/font-patcher) leveraging that script + the glyphs in this repo to hopefully make this process easier.

I have an [article](https://wayde.gg/articles/how-to-patch-a-font.html) explaining in detail how this works too. If you're on OSX and looking to patch a built-in, system font specifically, then this should be pretty helpful!

Wayde

**Comment by rodhash** [comment]
> Hello!
> 
> As someone who had struggled using the Nerd Fonts `font-patcher` script, I wrote a [tool](https://github.com/waydegg/font-patcher) leveraging that script + the glyphs in this repo to hopefully make this process easier.
> 
> I have an [article](https://wayde.gg/articles/how-to-patch-a-font.html) explaining in detail how this works too. If you're on OSX and looking to patch a built-in, system font specifically, then this should be pretty helpful!
> 
> Wayde

Hi @waydegg , this link to the article seems broken.

**Reply by Finii** [reply]
You should probably report that in their repo, i.e. https://github.com/waydegg/font-patcher/issues

I have no problem using the font-patcher on OSX, without that extra special additional script. Just make sure to use the `FontPatcher.zip` and do not try to use the script file stand alone (which is also mentioned in our main readme file).

Well, patching needs `fontforge`, but that can be easily installed on MacOS via Homebrew.
